### DISCUSSION TITLE
Change language of "Send chat to your email!" window

### DISCUSSION THREAD

**Discussion by Serag114** [LiveHelperChat/livehelperchat#1940] [Q&A]
Hello, I need some help with the Live Helper Chat.

As the title says, I'm trying to change the language of the "Send chat to your email!" modal window.
In my case, the header: "Send chat to your email!", the body text "We will send you a trascript of your chat with agent, so you can browse it later." and the text on the button "Send to email" need to be in another language.
I've changed all language settings I could find, and it works everywhere else, just not in this modal window.

Thanks in advance for the help

**Comment by netz0** [comment]
Make sure to clear the cache when changing language files and reload the page.
